Core/include/Acts/Propagator/EigenStepper.hpp

```diff
@@ -100,21 +101,12 @@ class EigenStepper {
       }
     }
 
-    /// Global particle position
-    Vector3D pos = Vector3D(0., 0., 0.);
-
-    /// Momentum direction (normalized)
-    Vector3D dir = Vector3D(1., 0., 0.);
+    /// Internal free vector parameters
+    FreeVector pars = FreeVector::Zero();
 
-    /// Momentum
-    double p = 0.;
-
-    /// The charge
+    /// The charge as the free vector can be 1/p or q/p
     double q = 1.;
```

Comment:
```suggestion
    int q = 1.;
```
Historically it was convenient to store it as a `double` since we need `q/p` calculated all the time. If you store `q/p` directly then we can change it to `int` (assuming that we never propagate quarks), right?

Reply:
I decided to keep it consistent with the return value of `parameters.charge()`

Reply:
Fixed.